Assets/Plugins/RootMotion/FinalIK.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Is this for something else or...

Reply:
Sorry about that, didn't mean to commit that!